docker/ubuntu/ubuntu20.Dockerfile

```diff
@@ -19,15 +19,16 @@ RUN apt-get install -y \
 
 ARG RELAY=v0.6.8
 
+ARG PHP=$(php -r 'echo substr(PHP_VERSION, 0, 3);')
```

Comment:
```suggestion
ARG PHP=$(php -r 'echo PHP_MAJOR_VERSION, ".", PHP_MINOR_VERSION;')
```

Reply:
This didn't work.

https://github.com/cachewerk/relay/pull/120